.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Update NPM
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: yarn install --frozen-lockfile
+      - run: yarn format
+      - run: yarn test:unit
+      - run: yarn test:int
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - run: yarn install --frozen-lockfile
+      - run: yarn publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

Comment:
@mmcc is this npm_token stored at the Github org level? Or does it have to be added to each repo?

I riffed off your work here: https://github.com/muxinc/upchunk/blob/master/.github/workflows/npm-publish.yml

Reply:
Just to follow up on this so it's in-line, yes, the npm token is now stored at the org level.